lib/smart_answer_flows/landlord-immigration-check.rb

```diff
@@ -6,6 +6,7 @@ def define
       status :published
```

Comment:
@pmanrubia I have updated commit notes with reference and links  from document[1] and flow diagram[2] 

[1] https://docs.google.com/document/d/1PtZxv9Fy-SOUtRSDvUq7_jbqLYUBn8Pp0nCf57U2UIE/edit
[2] https://docs.google.com/drawings/d/1x2547adffe616JMZ6qnI2ST5H2f-OnnUwSZ5isn5dzM/edit